types/proposal.go

```diff
@@ -72,10 +73,11 @@ func (p *Proposal) ValidateBasic() error {
 	if len(p.Signature) == 0 {
 		return errors.New("signature is missing")
 	}
-
-	if len(p.Signature) > MaxSignatureSize {
-		return fmt.Errorf("signature is too big (max: %d)", MaxSignatureSize)
-	}
+	// TODO(raneet10): ensure skipping signature check is innocuous.
+	// Also need to know how upstream would handle these checks for different keys (should be trivial but better to confirm).
+	// if len(p.Signature) > MaxSignatureSize {
```

Comment:
See my comment above for a similar hunk. We should try to enable an "if" here that makes sense (maybe checking the length based on signature type)

Reply:
For the sake of this fork does it matter given we only support one key type ?

Reply:
Resolved